emsize.py

```diff
@@ -37,32 +37,47 @@ def error(text):
 
 
 def parse_args(argv):
-  parser = argparse.ArgumentParser(description=__doc__)
-  parser.add_argument('-format', '--format')
-  parser.add_argument('file')
-  args = parser.parse_args(argv)
-  return args.file
+  files = []
+  params = []
+  appendFormat = True
+  ret = 0
+  for arg in argv:
+    if (arg.startswith('-')):
+      params.append(arg)
+    else:
+      files.append(arg)
+    if (arg.startswith('--format')):
+      appendFormat = False
+  if appendFormat:
+    params.append('--format=sysv')
+  for file in files:
```

Comment:
I guess the reason for handling the files manually (instead of just passing them all through to llvm-size together) is so we can handle the JS file separately. Did you consider collecting the list of files, pulling out the JS file, and then passing the rest through to llvm-size in a single invocation? Would there be any advantage to that?
If not, then whatever results in simpler code here is probably fine; I'm not too concerned about performance for this tool.